docs/installation.md

```diff
@@ -1,58 +1,81 @@
 # Installation Guide
 
-On this page we summarize the main installation options. For more details see [the main page of the repository](https://github.com/xournalpp/xournalpp#Installing). 
+On this page we summarize the main installation options. For more details see [the main page of the repository](https://github.com/xournalpp/xournalpp#Installing).
 
 Depending on your operation system you mainly have the following options to install Xournal++ on your device.
 
 ## Windows
+
 * Install the latest stable release (currently version 1.0.18) from the [releases page](https://github.com/xournalpp/xournalpp/releases)
+* Install the latest unstable release (automated nightly build) from the [releases page](https://github.com/xournalpp/xournalpp/releases/tag/nightly)
 * Bulid the latest development version (currently version 1.1.0+dev) from source as described on [this wiki page](https://github.com/xournalpp/xournalpp/blob/master/readme/WindowsBuild.md)
 
 ## MacOS
+
 * Install the latest stable release (currently version 1.0.18) from the [releases page](https://github.com/xournalpp/xournalpp/releases)
+* Install the latest unstable release (automated nightly build) from the [releases page](https://github.com/xournalpp/xournalpp/releases/tag/nightly)
 * Bulid the latest development version (currently version 1.1.0+dev) from source as described on [this wiki page](https://github.com/xournalpp/xournalpp/blob/master/readme/MacBuild.md)
 
-## Linux 
+## Linux
+
 ### Options for all/most distros
-The following options are available on _most major Linux distros_. 
 
-* Install the Snap Package (currently version 1.0.16) from the [official SnapCraft release](https://snapcraft.io/xournalpp)
+The following options are available on _most major Linux distros.
+
+* Install the Snap Package (currently version 1.0.16) from the [SnapCraft release](https://snapcraft.io/xournalpp)
```

Comment:
Should mention that the Snap is unofficial and unmaintained. I would also move this down since the other methods are preferred over the Snap..

Reply:
I agree. This is changed now (bullet points also).